CODING_GUIDELINES.md

```diff
@@ -139,6 +139,28 @@ Temporarily it may be allowed to split unit tests for a module into multiple `te
 
 Temporarily, tests for testing utilities can be placed next to the module containing them, with the name `test_util_<module>.py`. This should be taken as a hint that the tested utils should be moved into the library.
 
+#### Integration Tests Utils
+
+Integrations tests comprise their own utility functions for features' testing, found in `cases.py` for better test automation and simplification:
+
+    - Fields definitions, e.g. IJKField = Field[[IDim, JDim, KDim], np.int64]:
+    - Cases fixtures: cartesian_case for structured, e.g. IDim, JDim; unstructured_case, e.g. EdgeDim.
+    - Parameters allocations to objects, derived directly from the decorator's function definition:
```

Comment:
```suggestion
    - Parameter allocations for objects, derived directly from the decorator's function definition:
```

It's not very clear what parameter allocation actually means, at least not from the guidelines.